sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitGrid.java

```diff
@@ -69,7 +70,7 @@ public VisitGrid(VisitCriteria criteria) {
 
 		((Column<VisitIndexDto, Date>) getColumn(VisitIndexDto.VISIT_DATE_TIME))
 			.setRenderer(new DateRenderer(DateHelper.getLocalDateTimeFormat(I18nProperties.getUserLanguage())));
-		((Column<VisitIndexDto, String>) getColumn(VisitIndexDto.SYMPTOMATIC)).setRenderer(new BooleanRenderer());
+		((Column<VisitIndexDto, String>) getColumn(VisitIndexDto.SYMPTOMATIC)).setRenderer(new TextRenderer());
```

Comment:
You shouldn't need a renderer for this column because enums are automatically rendered according to their toString method.